packages/grafana-ui/src/components/ScrollContainer/ScrollContainer.mdx

```diff
@@ -0,0 +1,65 @@
+import { Meta, Preview, ArgTypes } from '@storybook/blocks';
+import { ScrollContainer } from './ScrollContainer';
+
+<Meta title="MDX|ScrollContainer" component={ScrollContainer} />
+
+# ScrollContainer
+
+This component is used to create a scrollable container. It uses native scrollbars, has an option to show scroll indicators, and supports most `Box` properties.
+
+## Migrating from CustomScrollbar
+
+ScrollContainer is intended to be a functionally equivalent replacement for CustomScrollbar. If you're not using any additional props in CustomScrollbar, it should be a drop-in replacement.
+
+If you were using additional props, there are roughly 3 categories of changes:
+
+1. Props that have been **renamed**
+1. Props that have been removed but the **same behavior can be achieved in a different way**
+1. Props that have been removed and are **no longer supported**
```

Comment:
```suggestion
1. Props that have been **renamed**
2. Props that have been removed but the **same behavior can be achieved in a different way**
3. Props that have been removed and are **no longer supported**
```

Reply:
it doesn't matter btw, markdown automatically sets the numbers anyway 😄